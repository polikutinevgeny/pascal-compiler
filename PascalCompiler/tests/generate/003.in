var a: Integer;
begin
    a := 5;
end.