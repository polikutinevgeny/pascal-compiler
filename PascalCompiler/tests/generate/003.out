.686p
include \masm32\include\masm32rt.inc
.mmx
.xmm
.code
main PROC
Push Ebp
Mov Ebp, Esp
Sub Esp, 4
Push 5
Pop Eax
Mov dword ptr [Ebp-4], Eax
Mov Esp, Ebp
Pop Ebp
exit
main ENDP
END main
